src/kernels/ADStressDivergence.C

```diff
@@ -15,6 +15,8 @@ InputParameters
 ADStressDivergence::validParams()
```

Comment:
Should this class be moved to MOOSE tensor mechanics instead of living in MALAMUTE? @cticenhour @lindsayad ? Is it worth making a separate issue, or has that code move been discussed and decided elsewhere already?

Reply:
I think having it upstream in tensor mechanics sounds ideal to me. Ultimately I would leave that decision to you mechanics folks though

Reply:
I agree with @lindsayad that it should be moved upstream. Since this is outside the scope of this PR, I've made an issue that we should address the status of this object - #113.